src/main/java/org/jenkinsci/plugins/GithubSecurityRealm.java

```diff
@@ -400,6 +411,23 @@ public HttpResponse doFinishLogin(StaplerRequest request)
         return HttpResponses.redirectToContextRoot();   // referer should be always there, but be defensive
     }
 
+    private void storeCredentials(String accessToken, GithubAuthenticationToken auth) throws IOException {
+        CredentialsStore credentialsStore = CredentialsProvider.lookupStores(Jenkins.getInstance()).iterator().next();
+        Credentials github = Iterables.find(credentialsStore.getCredentials(Domain.global()), new Predicate<Credentials>() {
+            @Override
+            public boolean apply(Credentials input) {
+                return input.getDescriptor().getId().equals("Github");
+            }
+        }, null);
+
+        UsernamePasswordCredentials credentials = new UsernamePasswordCredentialsImpl(CredentialsScope.USER, "Github", "Github Access Token", auth.getMyself().getLogin(), accessToken);
```

Comment:
@vivek assuming this Credential and the one you are creating during Blue Ocean's creation flow are the same, users authenticating via this plugin won't need to do anything.